packages/editor/src/store/private-selectors.js

```diff
@@ -49,3 +60,37 @@ export const getInsertionPoint = createRegistrySelector(
 export function getListViewToggleRef( state ) {
 	return state.listViewToggleRef;
 }
+const CARD_ICONS = {
+	wp_block: symbol,
+	wp_navigation: navigation,
+	page: pageIcon,
+	post: verse,
+};
+
+export const getPostIcon = createRegistrySelector(
+	( select ) => ( state, postType, area ) => {
```

Comment:
I don't like the "area" argument here, it's specific to a single post type, we could transform it to an optional "options" one which would be a bit better.